### ISSUE TITLE
Is concatination not supported?

### ISSUE THREAD

**Issue by mbcel** [f0cal/google-coral#26]
I am wondering if concatination in a model is not supported by the Google Edge TPU? 

For example when I do something like this in a model
```
cat = tf.concat([inFirst, inSecond], axis=-1)
output = tf.nn.relu6(cat, name='outputs')
```

And when I try to compile the .tflite file to an edgetpu file with the edgetpu_compiler it hangs and after a while aborts the compilation with 'Internal compiler error'. The docs say for concatination support limitations

> No fused activation function.
If any input is a compile-time constant tensor, there must be only 2 inputs, and this constant tensor must be all zeroes (effectively, a zero-padding op).<

What does this actually mean? To me this sounds like concatinating two tensors in the model is supported.

**Comment by Namburger** [comment]
@mbcel 
Concat is supported, just a little limited, there isn't enough info on the model for me to tell what's causing the compiler to aborts. I maybe able to check it out if you can attach it here.
